tscreen.go

```diff
@@ -653,6 +664,10 @@ func (t *tScreen) TPuts(s string) {
 	t.ti.TPuts(t.out, s, t.baud)
 }
 
+func (t *tScreen) TPutsRaw(s string) {
```

Comment:
I'm not sure having this be a public function buys anything.   IN fact, since all it does is wrap the call to io.WriteString(), and since there is only one call site for this function, I'd just inline it.